buildSrc/src/main/kotlin/com/datadog/gradle/config/AndroidConfig.kt

```diff
@@ -18,7 +18,7 @@ object AndroidConfig {
     const val MIN_SDK_FOR_COMPOSE = 21
     const val BUILD_TOOLS_VERSION = "33.0.0"
 
-    val VERSION = Version(1, 18, 0, Version.Type.SrBetaSnapshot(1))
+    val VERSION = Version(1, 18, 1, Version.Type.SrBetaSnapshot(1))
```

Comment:
it has is just that they are not visible as I merged them before :(. There are 3 fixes merged since 1.18.0. The idea is that I want to keep the minor version for fixes and the middle version for features.